modules/oadp-deletion-policy-1-2.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-deletion-policy-1-2_{context}"]
+= Deletion policy for OADP 1.2
+
+The Deletion Policy determines rules for removing data from a system, specifying when and how deletion occurs based on factors such as retention periods, data sensitivity, and compliance requirements. It manages data removal effectively while meeting regulations and preserving valuable information.
+
+== Deletion policy guidelines for OADP 1.2
+
+The following are the deletion policy guidelines for the OADP 1.2:
+
+* OADP 1.2.x Data Mover backup and restore require the Volume Snapshot Class `deletionPolicy` to be set to `Retain`.
```

Comment:
1. If setting `deletionPolicy` to `Retain` is mandatory, then I would request considering "....restore, you must set the `deletionPolicy` field to....."
2. IMO, it can be specified for clarity in the `VolumeSnapshotClass` CR, users can set the `deletionPolicy` field.